tools/depends/target/libinput/Makefile

```diff
@@ -0,0 +1,38 @@
+include ../../Makefile.include
+DEPS=../../Makefile.include Makefile
+
+# lib name, version
+LIBNAME=libinput
+VERSION=1.8.4
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.xz
+
+# configuration settings
+CONFIGURE=./configure --prefix=$(PREFIX) --enable-shared --disable-static --disable-documentation --disable-debug-gui --disable-tests --disable-libwacom --without-libunwind
```

Comment:
static

Reply:
libinput doesn't like to be static with a static libevdev https://bugs.freedesktop.org/show_bug.cgi?id=82292

Reply:
if it's just because the hard-coded static flag, patch it away

Reply:
I'm not going to do that. @whot did it for a reason https://cgit.freedesktop.org/wayland/libinput/commit/?id=f5e014f8b0c1d8a4d7bc94fd18aeb415f103fd75

Reply:
leaking symbols are fixed since libevdev 1.5.4 (this PR currently adds 1.5.7)

Reply:
leaking symbols are fixed in libevdev, *not* in libinput. linking libinput statically is not supported